client/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="shortcut icon" href="/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <link rel="manifest" href="/manifest.json" />
+    <title>Arlo, by VotingWorks</title>
+    <script type="text/javascript">
+      // Inject environment variables from the Flask backend by rendering this
+      // HTML page as a template.
```

Comment:
I simplified this logic (see the old public/index.html for what it was before). In dev, these injections won't work because index.html is rendered by vite, not Flask, but that's fine because I changed the rest of the code to only use them in production.